### DISCUSSION TITLE
Idea: Cycle through audio devices by double clicking on systray

### DISCUSSION THREAD

**Discussion by kenijo** [File-New-Project/EarTrumpet#711] [Ideas]
Thank for this really nicely executed product.

I have a request for managing audio devices:

- Could we have an option to cycle through the available audio devices by double clicking on the systray icon? (similar to [SoundSwitch](https://github.com/Belphemur/SoundSwitch))
![image](https://user-images.githubusercontent.com/5394591/87840238-7771aa00-c853-11ea-975e-977da09218c6.png)

- I would also like to extend this request to provide a setting to pre-select the devices the above functionality would cycle through.
![image](https://user-images.githubusercontent.com/5394591/87840254-8e180100-c853-11ea-9c95-dbfa3093dcef.png)

**Comment by sebastin25** [comment]
This is a great idea and if you can, add a option to configure hotkeys to cycle between devices

**Comment by simbesh** [comment]
This feature would allow me to uninstall SoundSwitch.
Agree hotkeys are a must
